### DISCUSSION TITLE
O que acham do meu primeiro package?

### DISCUSSION THREAD

**Discussion by Alecell** [frontendbr/forum#1877] [Discussão]
Galera vim compartilhar o meu primeiro projeto open source com vocês, eu to feliz pra camba que fiz isso (coisa que to a 2 anos pra fazer) e to mais feliz ainda que o projeto foi publicado ontem e ja tem 104 downloads!

O nome é Smart Routes, é um facilitador para organizar as rotas de uma aplicação, independe de framework, ele ajuda a organizar e manter de forma mais eficiente as rotas usadas na aplicação, tanto para declaração da rota no router quanto pra usar a rota quando for fazer um link.

O projeto está em TS, bastante no início ainda não tem todas as features de tipagem que o TS traria. Queria saber as críticas de vocês sobre o projeto, se puderem dar aquela fortalecida com PR's, starts no github ou só testar o package pra abrir uma issue vai ser super bem vindo!

[Github](https://github.com/Alecell/smart-routes)
[NPM](https://npmjs.com/package/smart-routes)

**Comment by dominguetigs** [comment]
@Alecell Não consigo acessar... dá 404 as páginas :-(

**Comment by Alecell** [comment]
> @Alecell Não consigo acessar... dá 404 as páginas :-(

Esqueci de colocar o `https://`, agora ta certo.
